### DISCUSSION TITLE
High CPU load on 2.7.7 (latest at this point in time) docker image

### DISCUSSION THREAD

**Discussion by jokakilla** [influxdata/influxdb#25176] [Q&A]
Hi guys,
I'm running influx on my home lab for almost two years now. It's running in a docker container on the official influx image with "latest" tag. After the last update I got 2.7.7 and got 70% CPU load after some hours. I've restarted the container but after two hours the CPU load sky rocketed again. This happened several times. Then I manually switched from latest to 2.7.6 and all is running fine again for over 24 hours now.

While the CPU load was very high there was no unusually intense load on the DB. After the first restart I've set log-level to debug but didn't notice anything unusual.

Has anybody else observed such behavior?

**Comment by wittmannchris** [comment]
Yes, same here.
